packages/block-library/src/table/editor.scss

```diff
@@ -16,4 +16,8 @@
 		box-shadow: inset 0 0 0 1px $blue-medium-500;
 		border-style: double;
 	}
+
+	&__cell-content {
```

Comment:
@jasmussen - do we have any pattern for the placeholder in the table cell?

Reply:
@jasmussen Just thought I'd ping you again on this question in case you missed it. Cheers.

Reply:
Hope you don't mind, I've also added you as a reviewer based on Grzegorz' feedback.

Reply:
Sorry I missed the ping! 

I think PR works well for me!

We don't have any pattern for placeholder text in tables, and I don't think we need it either. The table resembles a table in its natural state, and it's clear where you need to set the caret in order to type. Also, given table cells can grow very small, as this PR is about, it would be hard to add placeholder text that worked within those rules.

Reply:
Awesome - thanks for the reviews on this! 😃

Will merge now.